compositor/sessionmanager/sessionmanager.cpp

```diff
@@ -320,9 +285,6 @@ void SessionManager::requestHybridSleep()
     Q_EMIT hybridSleepRequested();
 }
 
-void SessionManager::cancelShutdownRequest()
-{
-    Q_EMIT shutdownRequestCanceled();
-}
+void SessionManager::cancelShutdownRequest() { Q_EMIT shutdownRequestCanceled(); }
 
-#include "moc_sessionmanager.cpp"
+#include "sessionmanager.moc"
```

Comment:
Yeah, it's not necessary in those cases. From the link you shared:

> The moc file may be included by the user in the C++ implementation file with a preprocessor #include. If it is not so included, it will be added to a separate file which is compiled into the target.